src/codeOfProgram/main/PlayingTheGame.java

```diff
@@ -0,0 +1,36 @@
+package codeOfProgram.main;
+
+import codeOfProgram.actions.AskingForAction;
+import codeOfProgram.resources.Configurations;
+
+public class PlayingTheGame {
+    public void playingGame(String nickname) {
+        int countOfPoints = 0;
+        boolean endOfGame;
+        boolean takeNewCard=false;
+
+        AskingForAction askingForAction = new AskingForAction();
+        askingForAction.askingForStartGame();
+        if(nickname==null){
+            nickname = askingForAction.askingForNickname();
+        }
+        DeckOfCards deckOfCards = new DeckOfCards();
+        ResultGame resultGame = new ResultGame();
```

Comment:
better name would be `gameResult` (I'd name it like this) or resultOfGame.

Reply:
+